src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -844,34 +834,22 @@ static unique_ptr<Scene> _sceneBuildHelper(SvgLoaderData& loaderData, const SvgN
             for (uint32_t i = 0; i < node->child.count; ++i, ++child) {
                 if (_isGroupType((*child)->type)) {
                     if ((*child)->type == SvgNodeType::Use)
-                        scene->push(_useBuildHelper(loaderData, *child, vBox, svgPath, depth + 1, isMaskWhite));
+                        scene->push(_useBuildHelper(loaderData, *child, vBox, svgPath, depth + 1));
                     else if (!((*child)->type == SvgNodeType::Symbol && node->type != SvgNodeType::Use))
-                        scene->push(_sceneBuildHelper(loaderData, *child, vBox, svgPath, false, depth + 1, isMaskWhite));
+                        scene->push(_sceneBuildHelper(loaderData, *child, vBox, svgPath, false, depth + 1));
                     if ((*child)->id) scene->id = djb2Encode((*child)->id);
                 } else if ((*child)->type == SvgNodeType::Image) {
-                    auto image = _imageBuildHelper(loaderData, *child, vBox, svgPath);
-                    if (image) {
+                    if (auto image = _imageBuildHelper(loaderData, *child, vBox, svgPath)) {
                         if ((*child)->id) image->id = djb2Encode((*child)->id);
                         scene->push(std::move(image));
-                        if (isMaskWhite) *isMaskWhite = false;
                     }
                 } else if ((*child)->type == SvgNodeType::Text) {
-                    auto text = _textBuildHelper(loaderData, *child, vBox, svgPath);
-                    if (text) {
+                    if (auto text = _textBuildHelper(loaderData, *child, vBox, svgPath)) {
                         if ((*child)->id) text->id = djb2Encode((*child)->id);
                         scene->push(std::move(text));
                     }
                 } else if ((*child)->type != SvgNodeType::Mask) {
-                    auto shape = _shapeBuildHelper(loaderData, *child, vBox, svgPath);
-                    if (shape) {
```

Comment:
@JSUYA @mgrudzinska Please confirm this logic.

Reply:
I remember it was for performance reasons.
(`isMaskWhite` comes from this PR. https://github.com/thorvg/thorvg/pull/1124)
I think it's okay to remove it if it's an unnecessary condition. (I didn't check the performance.)
I confirmed that it works normally when I tested the sample of https://github.com/thorvg/thorvg/pull/1130.